### ISSUE TITLE
Go to Implementations not available for generics interfaces

### ISSUE THREAD

**Issue by 1000p** [golang/vscode-go#2711]
### What version of Go, VS Code & VS Code Go extension are you using?
<details><summary>Version Information</summary><br>

* Run `go version` to get version of Go from _the VS Code integrated terminal_.
	- go1.19.3 darwin/amd64
* Run `gopls -v version` to get version of Gopls from _the VS Code integrated terminal_.
	- v0.11.0
* Run `code -v` or `code-insiders -v` to get version of VS Code or VS Code Insiders.
	- 1.76.1
* Check your installed extensions to get the version of the VS Code Go extension
	- v0.38.0
* Run Ctrl+Shift+P (Cmd+Shift+P on Mac OS) > `Go: Locate Configured Go Tools` command.
	- go:	/usr/local/go/bin/go: go version go1.19.3 darwin/amd64

	gotests:	/Users/mileniliev/Development/Go/bin/gotests	(version: v1.6.0 built with go: go1.19.3)
	gomodifytags:	/Users/mileniliev/Development/Go/bin/gomodifytags	(version: v1.16.0 built with go: go1.19.3)
	impl:	/Users/mileniliev/Development/Go/bin/impl	(version: v1.1.0 built with go: go1.19.3)
	goplay:	/Users/mileniliev/Development/Go/bin/goplay	(version: v1.0.0 built with go: go1.19.3)
	dlv:	/Users/mileniliev/Development/Go/bin/dlv	(version: v1.9.1 built with go: go1.19.3)
	staticcheck:	/Users/mileniliev/Development/Go/bin/staticcheck	(version: v0.3.3 built with go: go1.19.3)
	gopls:	/Users/mileniliev/Development/Go/bin/gopls	(version: v0.11.0 built with go: go1.19.3)
</details>

### Share the Go related settings you have added/edited

No settings added.

### Describe the bug
In short: If you have a generic interface, there is no way to check which type implements it.
[I will link a GoLand bug](https://youtrack.jetbrains.com/issue/GO-12702/Go-to-Declaration-Implementation-not-available-for-generics-interfaces) that describes the bug in detail. It appears that it's now sorted on their side.
This bug is frustrating to all codebases using generic interfaces.

### Steps to reproduce the behavior:
[Check steps to reproduce here](https://youtrack.jetbrains.com/issue/GO-12702/Go-to-Declaration-Implementation-not-available-for-generics-interfaces)

### Screenshots or recordings
N/A

**Comment by adonovan** [comment]
Thanks for reporting this bug. I've re-reported it in the gopls issue tracker here: https://github.com/golang/go/issues/59224.

**Comment by findleyr** [comment]
Thanks @adonovan. Closing this as a dupe of https://github.com/golang/go/issues/59224.

(in the future, we can actually just transfer this issue to the Go issue tracker, to preserve history).

**Comment by findleyr** [comment]
Thanks @adonovan. Closing this as a dupe of https://github.com/golang/go/issues/59224.

(in the future, we can actually just transfer this issue to the Go issue tracker, to preserve history).
